sdk/go/common/workspace/plugins_test.go

```diff
@@ -541,6 +541,32 @@ func TestPluginDownload(t *testing.T) {
 			assert.Equal(t, expectedBytes, readBytes)
 		})
 	})
+	t.Run("Test Downloading From GitLab Releases", func(t *testing.T) {
```

Comment:
optional nit: Go tests don't follow BDD-style naming conventions.
For subtests, you can omit all the context already present:
that it's a test and that it's a download test (because inside TestPluginDownload).
So this could just be "GitLab Releases" or "GitLab", and the test will still be called `TestPluginDownload/GitLab_Relases` for matching with `go test -run`.

```suggestion
	t.Run("GitLab Releases", func(t *testing.T) {
```

(I realize that other tests in the file are already following the pattern I'm advising against, but I figure it doesn't have to all be fixed in one go.)

Reply:
I'll fix up the lot